src/utils/exchange.js

```diff
@@ -55,7 +66,34 @@ export const getProviderLogo = (provider?: string) => {
       return zeroxLogo;
     case PROVIDER_CHANGELLY:
       return changellyLogo;
+    case PROVIDER_MOONPAY:
+      return moonPayLogo;
+    case PROVIDER_SENDWYRE:
+      return sendWyreLogo;
     default:
       return '';
   }
 };
+
+export const checkFiatProvider = (provider: string) => {
+  switch (provider) {
+    case PROVIDER_MOONPAY:
+    case PROVIDER_SENDWYRE:
+      return true;
+    default:
+      return false;
+  }
+};
+
+export const checkFiatCurrency = (symbol: string) => {
```

Comment:
Same here, I'd avoid "check" on names, we could use something like `isFiatCurrency` instead.

Reply:
Updated in last commit 👍